db/data_migrate/20180912144805_backfill_submission_entry_types.rb

```diff
@@ -0,0 +1,2 @@
+SubmissionEntry.sheet('InvoicesRaised').update(entry_type: 'invoice')
```

Comment:
Should this be `update_all` ?  I thought `update` only works for single resources..?

Reply:
so `update` will work on a relation and is different from `update_all` in that it will update each instance individually and trigger validations. `update_all` doesn't trigger validations. I figured it was safer to use `update` as we don't have a ton of entries and we'd see any issues there might be with the entries.